tests/st/libnetwork/test_mainline_multi_host.py

```diff
@@ -87,6 +89,9 @@ def test_multi_host(self):
             assert_number_endpoints(host1, 4)
             assert_number_endpoints(host2, 2)
 
+            # TODO - Semafore is too fast. Slow it down. Long term, let's add a retry...
```

Comment:
Can we not just add retries to the assert_connectivity method?  Easy to plumb through to the retry method.